src/Paket.Core/PackageResolver.fs

```diff
@@ -651,7 +651,7 @@ let Resolve (getVersionsF, getPackageDetailsF, groupName:GroupName, globalStrate
 
     let rec step (stage:Stage) (stackpack:StackPack) compatibleVersions (flags:StepFlags) =
 
-        let fuseConflicts currentConflict priorConflictSteps =
+        let inline fuseConflicts currentConflict priorConflictSteps =
```

Comment:
interesting. probably just because the stack size was halved due to inlining. but I'm fine with that!